src/python/bot/fuzzers/afl/launcher.py

```diff
@@ -257,9 +258,23 @@ class FuzzingStrategies(object):
   FAST_CAL_RANDOM_STRATEGY = 'strategy_fast_cal_random'
   FAST_CAL_MANUAL_STRATEGY = 'strategy_fast_cal_manual'
 
-  def __init__(self):
+  def __init__(self, target_path):
     strategy_pool = strategy_selection.generate_default_strategy_pool(
         strategy_list=strategy.AFL_STRATEGY_LIST, use_generator=False)
+
+    if environment.get_value('USE_RADAMSA'):
+      strategy_pool.add_strategy(strategy.CORPUS_MUTATION_RADAMSA_STRATEGY)
+      logs.log('Added radamsa to strategy pool.')
+
+    # Select a generator to use for existing testcase mutations.
+    self.generator = engine_common.select_generator(strategy_pool, target_path)
+    self.is_mutations_run = self.generator != engine_common.Generator.NONE
+
+    # Initialize to false. This is set to true if testcases are properly
+    # generated.
+    self.radamsa_generator = False
```

Comment:
are these actually getting piped into stats anywhere? I think you need to modify afl.stats.StatsGetter.set_strategy_stats  ?

Also, do we need to have these 2 variables? We could just have
self.generator_strategy = None # engine_common.Generator.VAL